src/devtools/client/webconsole/webconsole-ui.js

```diff
@@ -74,32 +69,6 @@ class WebConsoleUI {
     return this.proxy;
```

Comment:
There are a lot of references to this.proxy and this.getProxy() here.

Reply:
great. killing them